### ISSUE TITLE
missing Properties when right clicking on windows 10 taskbar after clicking Restored default settings in EP properties application

### ISSUE THREAD

**Issue by pyrates999** [valinet/ExplorerPatcher#3895]
Windows version: Windows 10 OS build 19045.5011
EP version: 22621.4317.67.1

In trying to troubleshoot why I could not enable weather, I clicked on Restore default settings in the EP Properties application.

Now when right clicking on the windows 10 taskbar, I no longer see the Properties option above Lock the taskbar.  I checked in Advanced and `Do not change the taskbar context menu (e.g. do not display the "Properties" item)` is not enabled.

Here is the screen shots of the settings I'm using:

![image](https://github.com/user-attachments/assets/9745a279-17d7-4209-81c0-eda74281051d)
![image](https://github.com/user-attachments/assets/6959f389-6c8b-468b-9251-5f31ca6b6e7f)
![image](https://github.com/user-attachments/assets/de11da98-4696-4d00-ba05-b31ddadcd89a)
![image](https://github.com/user-attachments/assets/582fa4e8-a45f-437f-9929-cad7a39e54e3)
![image](https://github.com/user-attachments/assets/a6244a59-f50e-43c7-a791-1c7a5239e246)
![image](https://github.com/user-attachments/assets/fc7ae6b1-4e2f-476c-ade3-df82ee79b328)
![image](https://github.com/user-attachments/assets/bf8327d3-4b2c-4b3a-a53e-f44aece1485f)
![image](https://github.com/user-attachments/assets/447c04b0-2bc7-4e73-af1b-0990018280cf)
![image](https://github.com/user-attachments/assets/de8f1e2d-23ea-4278-b55c-2b8974c7978b)
![image](https://github.com/user-attachments/assets/648dfbe8-c149-4913-9e60-bb84ccc92f3c)
![image](https://github.com/user-attachments/assets/935b707f-de5b-4e71-98b3-ad373419b333)

**Comment by MaxwellTheGoober** [comment]
Yeah,  the most recent windows updates have broken windows 10's EP, We'll just have to wait for an explorer patch

**Comment by Amrsatrio** [comment]
I'll fix it.

**Comment by 54145a** [comment]
same issue

**Comment by yonisade** [comment]
same here on Windows 11 24H2

**Comment by pyrates999** [comment]
This has been fixed for me on EP release 26100.4946.69.4: https://github.com/valinet/ExplorerPatcher/releases

My windows build is: windows 10 22h2 19045.6332.

> same here on Windows 11 24H2

That isn't detailed enough.  Please post windows version and build.  You can find this by running `winver`.  Please include if you are running x86-64 or Arm.  And what language your windows is in.

Update your EP to the same version as mine if it is not.

**Comment by yonisade** [comment]
My build is: Windows 11 24H2 26100.6725

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
`C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI`

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.

If it doesn't show up now, or doesn't work, click on restart file explorer in the bottom left in the EP properties application.

You can also try to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe.

Also try restarting your computer.

---

This is now working for me when I enable and disable the above option.  I did not need to restart file explorer, nor did I need to restart my computer.  But you may have to.

Please show a screen shot of what you get when you right click on the taskbar with the above option disabled.

**Comment by yonisade** [comment]
Didn't help, tried all the steps you suggested.
<img width="964" height="536" alt="Image" src="https://github.com/user-attachments/assets/73aab96b-43f7-42bd-b4af-8473ebd76080" />

**Comment by pyrates999** [comment]
https://github.com/valinet/ExplorerPatcher/wiki/All-features#disable-the-windows-11-context-menu
